packages/amplify_storage_s3/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/types/FlutterDownloadFileRequest.kt

```diff
@@ -47,15 +48,14 @@ data class FlutterDownloadFileRequest(val request: Map<String, *>) {
     }
 
     companion object {
-        fun isValid(request: Map<String, *>): Boolean {
-            var valid = true
-            if (request["path"] !is String) {
-                valid = false
+        private const val validationErrorMessage: String = "Download request malformed."
+        fun validate(request: Map<String, *>) {
+            if(request["path"] !is String? || request["path"] == null) {
+                throw AmplifyFlutterValidationException(validationErrorMessage, "Ensure that path variable is present.")
```

Comment:
Nit: can you replace variable with attribute?

Reply:
Done